src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Extends Command class
+ * When executed, calls Ui method to print output
+ */
+public class AddCommand extends Command {
+    private Task task;
+
+    /**
+     * Constructor
+     * Initialise task attribute to the input task to be added
+     * @param task Task to be added
+     */
+    public AddCommand (Task task) {
+        this.task = task;
+    }
+
+    /**
+     * Handles adding of task to tasklist, calling of ui method to print outpyt and ypdating storage
+     * @param taskList TaskList has all current tasks
+     * @param ui Ui handles printing to output
+     * @param storage Storage saves tasklist
+     * @return String output from ui
+     */
```

Comment:
```suggestion
    /**
     * Handles adding of task to tasklist, calling of ui method to print output and updating storage.
     *
     * @param taskList TaskList has all current tasks.
     * @param ui Ui handles printing to output.
     * @param storage Storage saves tasklist.
     * @return String output from ui.
     */
```
For this comment (and other comments as well), there should be a spacing between the description and parameter section. In addition, there should be a punctuation behind each parameter description. 
Other than that, there are small typos for the words "output" and "updating".